packages/web/src/components/spa/HackAUBG/VideoSection/video_section.css

```diff
@@ -24,20 +27,28 @@
     justify-content: center;
     align-items: center;
     display: flex;
-    padding: 3% 0;
     padding: 3% 0%;
     text-align: justify;
 }
 
 @media only screen and (max-width: 900px) {
+    .video-container {
+        height: 90vh;
+    }
+
     .video-header {
-        text-size: 0.5rem;
-        text-align: justify;
+        font-size: 1.5em;
```

Comment:
Check the font-size immediate change between 1200px and 1300px. Please make the font-size bigger for devices smaller than 1300px